selfdrive/car/honda/carstate.py

```diff
@@ -203,6 +207,11 @@ def update(self, cp, cp_cam, cp_body):
     ret.vEgoRaw = (1. - v_weight) * cp.vl["ENGINE_DATA"]["XMISSION_SPEED"] * CV.KPH_TO_MS * self.CP.wheelSpeedFactor + v_weight * v_wheel
     ret.vEgo, ret.aEgo = self.update_speed_kf(ret.vEgoRaw)
 
+    if self.CP.carFingerprint in HONDA_BOSCH:
+      ret.vEgoCluster = cp.vl["CAR_SPEED"]["ROUGH_CAR_SPEED_2"] * CV.KPH_TO_MS
```

Comment:
```suggestion
      ret.vEgoCluster = cp.vl["CAR_SPEED"]["ROUGH_CAR_SPEED_2"] * CV.KPH_TO_MS
```

Conversion based on `["CAR_SPEED"]["IMPERIAL_UNIT"]`?